test/cli/test-bitswap.js

```diff
@@ -41,42 +39,32 @@ describe('bitswap', function () {
       httpAPI.stop(done)
     })
 
-    it('wantlist', (done) => {
+    it('wantlist', () => {
       const api = httpAPI.server.select('API')
 
       api.inject({
         method: 'GET',
         url: `/api/v0/block/get?arg=${key}`
-      }, (res) => {})
+      }, () => {})
 
-      nexpect.spawn('node', [process.cwd() + '/src/cli/bin.js', 'bitswap', 'wantlist'], {env})
-        .run((err, stdout, exitcode) => {
-          expect(err).to.not.exist
-          expect(exitcode).to.equal(0)
-          expect(stdout).to.be.eql([
-            key
-          ])
-          done()
-        })
+      return ipfs('bitswap wantlist').then((out) => {
+        expect(out).to.be.eql(key)
+      })
```

Comment:
Look closely, the `inject` call is just so we get the right response, but the actual command shouldn't happen inside, otherwise it would hang forever.